app/views/users/mailer/confirmation_instructions.html.erb

```diff
@@ -1,5 +1,6 @@
-<p>Welcome <%= @email %>!</p>
+<p><%= t('.greeting', :recipient => @resource.email, :default => "Welcome #{@resource.email}!") %></p>
```

Comment:
why is this default here in the code? I would have expected this string in the English locales and the English locales set as the fallback locale. This should have the same effect but the code would be much cleaner